clippy_lints/src/unnecessary_struct_initialization.rs

```diff
@@ -38,7 +38,7 @@ declare_clippy_lint! {
     /// moved out of, see [#10547](https://github.com/rust-lang/rust-clippy/issues/10547).
     ///
     /// Empty structs are ignored by the lint.
-    #[clippy::version = "1.70.0"]
+    #[clippy::version = "1.82.0"]
```

Comment:
The version was correct I think. The lint has definitely existed for longer than this version at least. It was only extended to catch more patterns recently

Reply:
Right, there was something confusing where a changelog entry said it added a lint and that was the only one with changes. On the plain I didn't have WIFI to check the PR ^^

Should be correct now